legion/engine/core/filesystem/view.hpp

```diff
@@ -144,6 +146,12 @@ namespace legion::core::filesystem
         mutable navigator::solution m_foundSolution{};
     };
 
+    class view_util
+    {
+    public:
```

Comment:
this is created as a class but is named like a struct (view_util instead of ViewUtil). I'm also uncertain why this is created for a single function (get_view_path) which seemingly should just be a part of `view`?

Reply:
hmmm i don't even know this class, i'll have a look at this